node/tessel-export.js

```diff
@@ -916,6 +916,10 @@ Tessel.LED.prototype.toggle = function(callback) {
   this.write(this.value ? 0 : 1, callback);
 };
 
+Tessel.LED.prototype.isOn = function() {
+  return this.value === 1;
+};
+
```

Comment:
@rwaldron Don't know if you saw this since I edited, but what do you think?

Actually, I might even consider it a bit strange to have `is*` for getting and `*` for setting. We could make all of the following work:

```javascript
var led = ...

if (led.on) console.log('The light is on')
if (led.off) console.log('The light is off')

led.on = true // Now the light is on
led.off = true // Now the light is off

led.on = false // Now the light is off
led.off = false // Now the light is on

led.toggle() // Toggle the LED state
```